core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -134,7 +134,7 @@ private SqlLibraryOperators() {
 
   /** The "DATE_ADD(date, numDays)" function
    * (Spark) Returns the date that is num_days after start_date. */
-  @LibraryOperator(libraries = {SPARK})
+  @LibraryOperator(libraries = {SPARK, ORACLE, HIVE})
```

Comment:
You enable this in Oracle, do we have a test?

Reply:
delete oracle~ it is my mistake.